src/NServiceBus.Core.Tests/Serializers/Json/JsonMessageSerializerTest.cs

```diff
@@ -9,8 +9,33 @@ namespace NServiceBus.Serializers.Json.Tests
     using MessageInterfaces.MessageMapper.Reflection;
     using NUnit.Framework;
 
+    class MessageWithException
+    {
+        public Exception Blah { get; set; }
```

Comment:
It would blow for any message with a prop that is an interface + contains methods. I'll try to change the test!